### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: src/file_browser/mod.rs
Comment (baco) [inline src/file_browser/mod.rs:215]: What does the `strong` attribute does? Couldn't find it in the docs.

Comment (sdroege) [reply src/file_browser/mod.rs:215]: Same as `@strong` before, and what the custom `clone!` macro here did previously without any attributes: It calls `store.clone()` and moves that into the closure.

See https://gtk-rs.org/gtk-rs-core/stable/latest/docs/glib/macro.clone.html#passing-a-strong-reference

Comment (baco) [reply src/file_browser/mod.rs:215]: LGTM

**Review by baco — APPROVED** [review]

**Comment by Lyude** [comment]
BTW - do either of you two want maintainer access?

**Comment by baco** [comment]
> BTW - do either of you two want maintainer access?

🤔 Well... that really depends on whether are you going to stick around or not.

Regarding myself I have only an Advent of Code experience in Rust. I am willing to learn more of it, but I definitely lack the skills to build a project like this from the ground up.

I have played myself with Python GTK applications in the past, using Glade, but I gather it's not the same.

So, I can give a hand, I will try to improve in Rust, but I'll continue needing some mentoring.

**Comment by Lyude** [comment]
That's totally fair haha, FWIW this was actually my project for learning rust (definitely why there are some design choices I made which I now regret lol). To be clear - I'm definitely sticking around, I still use nvim-gtk pretty much daily at my job. I just haven't dedicated as much spare time towards working on it since I've been getting my fill of rust working on writing up KMS bindings for rust in the Linux kernel :P. I would eventually like the time to do some improvements though (getting rid of tokio now that I know more about rust and understand it's not needed, rewriting our text renderer again to make use of some of the new GTK interfaces for fixed width text rendering, getting the flatpak actually up and published, etc.).
I don't mind if you hold off on maintainership though - don't want to put the pressure of reviewing stuff on you if you're not always sure what to look for :)

**Comment by sdroege** [comment]
I don't really have time for yet another project, but I can definitely help reviewing *some* PRs and doing things like this PR in the future. I'm using nvim-gtk as my main editor so have some interest in keeping it running at least :)

**Pull request merged**
